src/Controls/src/Core/Shapes/EllipseGeometry.cs

```diff
@@ -56,8 +56,8 @@ public double RadiusY
 		/// <include file="../../../docs/Microsoft.Maui.Controls.Shapes/EllipseGeometry.xml" path="//Member[@MemberName='AppendPath']/Docs" />
 		public override void AppendPath(PathF path)
 		{
-			var centerX = (float)RadiusX;
-			var centerY = (float)RadiusY;
+			var centerX = (float)Center.X;
```

Comment:
:astonished:

Reply:
This https://github.com/dotnet/maui/issues/7460 issue still remains in Android

Unzip the below Project 
[BorderClipImage.zip](https://github.com/dotnet/maui/files/9216849/BorderClipImage.zip)

**Screenshots**

See the clipping issues in Android

In Android 
<img width="188" alt="image" src="https://user-images.githubusercontent.com/102580874/181697605-6be5ebfb-fcec-4d06-9fe5-f0b3ff31da8a.png">
In Windows
<img width="701" alt="image" src="https://user-images.githubusercontent.com/102580874/181697761-d3b4d78a-4eab-4362-974c-472f0d1c850e.png">